eng/pipelines/templates/jobs/archetype-sdk-client.yml

```diff
@@ -132,8 +132,7 @@ jobs:
           - pwsh: |
               $outputPath = Join-Path -Path $(Build.ArtifactStagingDirectory) packages/${{ artifact.Name }}/package-info.json
 
-              # TODO: Read version from a .h file instead of version.txt
-              $version = Get-Content -Path sdk/${{ parameters.ServiceDirectory }}/${{ artifact.Path }}/version.txt
+              $version = eng/scripts/Get-PkgVersion -ServiceDirectory ${{ parameters.ServiceDirectory }} -PackageName ${{ artifact.Path }}
```

Comment:
Does PS allow you to call a script without an extension?

Reply:
This is a typo, however it does work.